### DISCUSSION TITLE
代理套Cloudflare CDN，部分站点无法访问的原因可能有？

### DISCUSSION THREAD

**Discussion by Zi-yun2333** [XIU2/CloudflareSpeedTest#544] [提问] [answered]
比如Pixiv，Chat GPT，Cloudflare是打不开的。但是奈飞，Facebook这种又问题。用TLS协议走域名443端口伪装的。
所以只能作为魔法备用方案吗，但是它真的很快。优选ip之后下行可以跑到800兆。>.<

**Comment by Zi-yun2333** [comment]
如图，唔
![20240923-181020](https://github.com/user-attachments/assets/649de8c5-4cc9-4665-83a8-30d250675424)
![20240923-181057](https://github.com/user-attachments/assets/2121e5af-64a8-4a4f-8df1-a6fcaba5ecf8)
![20240923-181121](https://github.com/user-attachments/assets/234832f4-5ceb-447b-a098-154467971c24)

**Comment by XIU2** [comment, marked as answer]
对于代理套 Cloudflare CDN 的使用方式，因为我没用过，所以也不太清楚你这个问题具体原因是什么。

不过呢，我的印象中，代理套 Cloudflare CDN 其实就是在你与代理服务器之间加了个中转（就是 CF CDN 的节点服务器）：

```yaml
# 代理套了 Cloudflare CDN：
你通过代理软件访问某个网站 <=> Cloudflare CDN <=> 代理服务器 <=> 目标网站（如你说的 Pixiv、ChatGPT、Cloudflare 等）

# 直连代理服务器：
你通过代理软件访问某个网站 <=> 代理服务器 <=> 目标网站（如你说的 Pixiv、ChatGPT、Cloudflare 等）
```

因此理论上，不管你是直连代理还是走套了 Cloudflare CDN 的代理，你的落地 IP 都是代理服务器的 IP，目标网站看到的 IP 依然是你的代理服务器的 IP，Cloudflare CDN 只负责了中转流量（流量也是代理软件加密后的），按理说是影响不到代理软件访问结果的。

另外，你提的这几个网站示例（Pixiv、ChatGPT、Cloudflare）全都是使用了 Cloudflare CDN 的，这个问题特征可能是一个突破点？

不过，这种问题可能更适合在专门讨论代理技术相关的网站询问，这项目自带的论坛人太少，也基本都是搞自选优选的，而你的问题并不是由自选优选引起的。。。
